sql_generators/glean_usage/event_monitoring_live.py

```diff
@@ -50,6 +60,9 @@ def _get_tables_with_events(self, v1_name: str) -> Set[str]:
                 latest_history = metric.get("history", [])[-1]
                 pings.update(latest_history.get("send_in_pings", []))
 
+        if bq_dataset_name in self._get_prod_datasets_with_event():
+            pings.add("events")
+
```

Comment:
```suggestion

```

The materialized views only need to monitor tables that have event-type probes as defined in probe-info. Some apps don't seem to have such probes in the `events` tables, so we shouldn't add them here since not monitoring them reduces cost a bit.

Reply:
Ok so do we only care about custom events defined in the app and not in the libraries?  In that case, https://github.com/mozilla/bigquery-etl/pull/5798 would be at least partially correct and a bunch of materialized views would need to be deleted to get what's deployed consistent with the generated sql.  I didn't think it made sense to suddenly delete them.

If we do care about events in libraries then I think they all will get at least `glean.restarted` in the events ping because of glean-core.  If it's more complicated than that then I can leave this for now since nothing is broken at the moment.

Reply:
Yeah, good point. I guess adding `events_v1` to the monitoring shouldn't cause any issues of stuff breaking.